smallrye-reactive-messaging-amqp/src/main/java/io/smallrye/reactive/messaging/amqp/tracing/AmqpAttributesExtractor.java

```diff
@@ -39,7 +38,7 @@ private static class AmqpMessagingAttributesGetter implements MessagingAttribute
         // Required
         @Override
         public String system(final AmqpMessage<?> amqpMessage) {
-            return AmqpConnector.CONNECTOR_NAME;
+            return "AMQP 1.0";
```

Comment:
I am wondering if this should be `amqp` only?

Reply:
I hesitated on that as well, then kept the one we had before.

Reply:
Ok, just leave that as is :)

Reply:
AMQP is misleading. For most people it's the AMQP from RabbitMQ (8. something), and not AMQP 1.0. So, better be clear about that.

Reply:
We can set the system name and the protocol. For RabbitMQ we had `amqp` set as the protocol, but we cleared that out.

For AMQP, I guess that both the system and protocol are `amqp`.